### PR TITLE
Translated the whole chapter 4 into Thai

### PR DESCRIPTION
**Pull request by jomariya23156**
Hi, I have finished translating the whole chapter 4 into Thai. Please check it out and let me know if there is anything wrong. Cheers!

Related to #64

### CODE REVIEW

File: chapters/th/_toctree.yml [hunk 1]
```diff
@@ -6,4 +6,20 @@
...
   sections:
   - local: chapter1/1
-    title: บทนำ
\ No newline at end of file
+    title: บทนำ
+
+- title: 4. การแบ่งปันโมเดลและ tokenizers
+  sections:
+  - local: chapter4/1
+    title: Hugging Face Hub
```
Comment (lewtun) [inline chapters/th/_toctree.yml:14]: Small suggestion to keep the title the same as the English counterpart

```suggestion
    title: The Hugging Face Hub
```

**Review by lewtun — APPROVED** [review]
Thank you for this incredible contribution @jomariya23156 - it's so cool to see a whole chapter translated in its entirety 🥳 !

I'll ask the Thai speakers for a quick review, but the structure of the files looks great to me

File: chapters/th/chapter4/6.mdx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+/>
+
+### 5. วัตถุ (objects) ใดต่อไปนี้ของ 🤗 Transformers library ที่สามารถถูกแบ่งปันได้โดยตรงบน Hub ผ่านคำสั่ง `push_to_hub()`?
+
+{#if fw === 'pt'}
+<Question
+	choices={[
+		{
+			text: "tokenizer",
+			explain: "ถูกต้องแล้ว! tokenizer ทั้งหมดมีคำสั่ง <code>push_to_hub</code> และใช้มันในการดันไฟล์ทั้งหมดของ tokenizer (คำศัพท์ (vocabulary), สถาปัตยกรรมของ tokenizer และอื่นๆ) ไปยัง repo ที่กำหนดได้ แต่นี่ก็ไม่ใช่คำตอบที่ถูกเพียงข้อเดียว!"
```
Comment (lewtun) [inline chapters/th/chapter4/6.mdx:112]:
```suggestion
			explain: "ถูกต้องแล้ว! tokenizer ทั้งหมดมีคำสั่ง <code>push_to_hub</code> และใช้มันในการดันไฟล์ทั้งหมดของ tokenizer (คำศัพท์ (vocabulary), สถาปัตยกรรมของ tokenizer และอื่นๆ) ไปยัง repo ที่กำหนดได้ แต่นี่ก็ไม่ใช่คำตอบที่ถูกเพียงข้อเดียว!",
```

File: chapters/th/chapter4/6.mdx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+            correct: true
+		}
+	]}
+/>
+{:else}
+<Question
+	choices={[
+		{
+			text: "tokenizer",
+			explain: "ถูกต้องแล้ว! tokenizer ทั้งหมดมีคำสั่ง <code>push_to_hub</code> และใช้มันในการดันไฟล์ทั้งหมดของ tokenizer (คำศัพท์ (vocabulary), สถาปัตยกรรมของ tokenizer และอื่นๆ) ไปยัง repo ที่กำหนดได้ แต่นี่ก็ไม่ใช่คำตอบที่ถูกเพียงข้อเดียว!"
```
Comment (lewtun) [inline chapters/th/chapter4/6.mdx:137]:
```suggestion
			explain: "ถูกต้องแล้ว! tokenizer ทั้งหมดมีคำสั่ง <code>push_to_hub</code> และใช้มันในการดันไฟล์ทั้งหมดของ tokenizer (คำศัพท์ (vocabulary), สถาปัตยกรรมของ tokenizer และอื่นๆ) ไปยัง repo ที่กำหนดได้ แต่นี่ก็ไม่ใช่คำตอบที่ถูกเพียงข้อเดียว!",
```

**Review by lewtun — COMMENTED** [review]
FYI @jomariya23156 I just noticed two places where we need a comma in the quizzes - would you mind fixing them please? (I marked them in the PR)

**Comment by jomariya23156** [comment]
@lewtun Thanks for letting me know! I have fixed them :)

**Pull request merged**